### PR TITLE
start of veANGLE docs

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

**Comment by tutacrypto** [comment]
Still lacking the link for the 2nd article of the upgrade series before merging

File: governance/veANGLE/boost.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+\end{equation}
+$$
+
+$$
+\begin{equation}
+\texttt{share of veANGLE supply} \geq  \texttt{share of liquidity in the pool}
+\end{equation}
+$$
+
+**As we can see, an address having a higher share of the total veANGLE supply than its share of liquidity provided in the pool for which the boost is being computed will be considered as having x2.5 more liquidity than an address with no veANGLE.**
```
Comment (sogipec) [inline governance/veANGLE/boost.md:68]: Duplicate "as we can see" and we should avoid "we"

File: governance/veANGLE/boost.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+From these formulas, we conclude that to get the maximum rewards boost, an address needs to hold at least similar share of the veANGLE total supply than its share of the pool.
+
+#### Examples
+
+Now that we understand how veANGLE holders get a boost on capital compared to non-holders, let’s look at how this translates into an **actual boost on rewards received.**
```
Comment (sogipec) [inline governance/veANGLE/boost.md:74]: I think we should avoid "we" in docs. The style should be a bit more formal than articles

File: governance/veANGLE/boost.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+**Example 1:**
+
+O*ne holder of veANGLE with a big share of the pool.*
```
Comment (sogipec) [inline governance/veANGLE/boost.md:78]: Weird here O*ne

File: governance/veANGLE/interests.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Interests redistribution
+
+### Share of interests for veANGLE holders
+
+The share of interests redistributed to veANGLE holders is a parameter that can be set and updated by Governance. The goal is that this parameter is high enough to bring added value to veANGLE holders, but not too high to damage the reserves being built by the protocol and the yield earned by SLPs.
+
+### Redistribution
+
+The protocol will distribute interests in the form of Angle tokens, beginning by redistributing sanUSDC_EUR.
```
Comment (sogipec) [inline governance/veANGLE/interests.md:21]: We should also explain here the redistribution flow: like how it's going to work for FEI, FRAX and DAI -> converted to USDC and then deposited

File: governance/veANGLE/interests.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+In practice, if the redistribution of interests to veANGLE token holder is at 30%, here is what is going to happen:
+
+- 30% of all interests generated will be converted to USDC
+- the USDC will be deposited into the protocol against sanUSDC_EUR
+- the sanUSDC_EUR will be redeemable by veANGLE holders
+- After having redeemed their sanUSDC_EUR, veANGLE holders will be able to withdraw USDC from the protocol, or keep sanUSDC_EUR and enjoy the interests generated for SLPs.
```
Comment (sogipec) [inline governance/veANGLE/interests.md:28]: Let's avoid this last sentence, from a legal perspective better not to say it

File: governance/veANGLE/voting.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+veANGLE holders will be the main voting power of the Angle protocol. As such, they will be shaping the future of the protocol.
+
+To facilitate the voting process, the governance module of the protocol will change after the upgrade. Votes will still happen on Snapshot, and will then get executed by a multi-sig. This multi-sig will be composed of a mix of core team members and external people, including public figures if possible (anon or not) to increase accountability.
+
+## Voting on gauge weights
+
+veANGLE holders are also the ones voting on how the weekly ANGLE rewards are distributed between the different staking contracts (called gauges after the upgrade).
+
+This vote will be held weekly following the current mining liquidity program.
```
Comment (sogipec) [inline governance/veANGLE/voting.md:19]: Vote will not be held weekly, like Snapshot on the vote is going to be taken weekly but people will only be able to update their gauge votes every two weeks

**Comment by sogipec** [comment]
Would add several points beyond the comments:
- veANGLE Governance Whitelisting
- More details on gauge weights (like here): https://docs.frax.finance/vefxs/gauge

**Pull request merged**